Sources/View/UICollectionView+Reusable.swift

```diff
@@ -93,9 +93,9 @@ public extension UICollectionView {
    except when your type is in a variable and cannot be determined at compile time.
    - seealso: `dequeueReusableSupplementaryViewOfKind(_:,withReuseIdentifier:,forIndexPath:)`
    */
-  final func dequeueReusableSupplementaryView<T: UICollectionReusableView where T: Reusable>
-    (elementKind: String, indexPath: NSIndexPath, viewType: T.Type = T.self) -> T {
-    let view = self.dequeueReusableSupplementaryViewOfKind(elementKind, withReuseIdentifier: viewType.reuseIdentifier, forIndexPath: indexPath)
+  final func dequeueReusableSupplementaryView<T: UICollectionReusableView>
+    (ofKind elementKind: String, indexPath: IndexPath, viewType: T.Type = T.self) -> T where T: Reusable {
```

Comment:
I think the new API for this should be `dequeueReusableSupplementaryView(ofKind:for:viewType:)` instead?